components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -175,6 +188,27 @@ assertTrue(Flags.a11yEnabled.testHasValue())
 assertTrue(Flags.a11yEnabled.testGetValue())
 ```
 
+## Labeled Booleans
+
+Used to record multiple related truth values
+
+For example you may want to find which URI schemes are used in your application, like `"http"`, `"https"`, `"about"`:
+
+**TODO: What's the typical use-cases here? Should we instead really do a kind of "label set" metric?**
```

Comment:
What do you mean by "label set"?

Reply:
A set of unique string values (vs. a string list with non-uniques). E.g.:
```
val x: StringSetMetric
x.add("http")
x.add("http")
x.add("about")
// x now contains: {"http", "about"}
```

Reply:
Ah, good point. Can you please file a bug?

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1557353